File: docs/source/_src_snippets/simulation/environment_setup/environment_models/simple_rotation_model.py
initial_orientation = ...

initial_time = ...
rotation_rate = ...

original_frame = "J2000"
target_frame = "IAU_Earth"

body_settings.get_body( "Earth" ).rotation_model_settings = environment_setup.rotation_model.simple( original_frame, target_frame, 
	initial_orientation, initial_time, rotation_rate)

File: docs/source/_src_snippets/simulation/environment_setup/environment_models/simple_rotation_model.cpp
Eigen::Quaterniond initialOrientation = ...

double initialTime = ...
double rotationRate = ...

std::string originalFrame = "J2000";
std::string targetFrame = "IAU_Earth";

bodySettings[ "Earth" ]->rotationModelSettings = std::make_shared< SimpleRotationModelSettings >(
    originalFrame, targetFrame , initialOrientation, initialTime, rotationRate );